src/haddock/modules/refinement/flexref/cns/flexref.cns

```diff
@@ -321,12 +321,12 @@ evaluate ($failure = $result)
 
 
 {* Define initial weights for distance restraints ============================== *}
-evaluate ($unamb_scale = $Data.unamb_hot)
+evaluate ($unambig_scale = $Data.unamb_hot)
```

Comment:
shouldn't then also unamb_hot, unamb_cool... and related be modified for consistency?

Reply:
extremely good point here !